docs/index.md

```diff
@@ -35,15 +37,9 @@ print(spec.model_dump())
             'dtype': '|u1',
             'fill_value': 0,
             'order': 'C',
-            'filters': None,
+            'filters': [],
```

Comment:
For whoever reviews this: I updated `filters` to be an empty list instead of `None` if there are no filters. This matches `zarr-python` 3 behaviour. I would be easy enough to revert to previous behaviour though, and set this back to `None`. Would appreciate thoughts on this.

Reply:
if the goal is to accurately model what's in zarr-python 2 metadata documents, then `filters: []` [is invalid](https://github.com/zarr-developers/zarr-specs/blob/1d0989e9d02ff7d6671e324f6bf17e3702181662/docs/v2/v2.0.rst?plain=1#L77-L79)

Reply:
I will definitely fix this to revert to `filters=None` then 👍

Reply:
fwiw I think the spec is wrong here, and it should allow `filters: []`, but it is what it is 🤷

Reply:
Fixed